### ISSUE TITLE
Plugin Settings GUI support (w/ attached attempt)

### ISSUE THREAD

**Issue by jmissig** [smockle/homebridge-lutron-caseta#46]
New-style plugins can be configured in [a settings GUI](https://github.com/oznu/homebridge-config-ui-x/wiki/Developers:-Plugin-Settings-GUI).

I made a `config.schema.json`. It works for me. Haven’t really verified it beyond that.

[config.schema.json.zip](https://github.com/smockle/homebridge-lutron-caseta/files/4555979/config.schema.json.zip)

```
{
   "pluginAlias":"LutronCasetaPlatform",
   "pluginType":"platform",
   "singular":true,
   "schema":{
      "type":"object",
      "properties":{
         "bridgeConnection":{
            "type":"object",
            "properties":{
               "host":{
                  "type":"string",
                  "required":true,
                  "format":"hostname"
               }
            }
         },
         "accessories":{
            "type":"array",
            "items":{
               "title":"Pico Remote Config",
               "type":"object",
               "properties":{
                  "name":{
                     "type":"string",
                     "required":true
                  },
                  "type":{
                     "type":"string",
                     "oneOf":[
                        { "title":"PJ2-2B", "enum":["PJ2-2B"] },
                        { "title":"PJ2-3B", "enum":["PJ2-3B"] },
                        { "title":"PJ2-3BRL", "enum":["PJ2-3BRL"] },
                        { "title":"PJ2-4B", "enum":["PJ2-4B"] },
                        { "title":"PICO-REMOTE", "enum":["PICO-REMOTE"] }
                     ],
                     "required":true
                  },
                  "integrationID":{
                     "type":"integer",
                     "required":true
                  }
               }
            }
         }
      }
   }
}
```

**Comment by smockle** [comment]
@jmissig If you open a PR, I’d be happy to merge it and publish a new version with this included.

**Comment by smockle** [comment]
Fixed by #47

**Comment by smockle** [comment]
Published version `3.0.6`, which includes these changes. I verified that the `config.schema.json` is included in the package: https://unpkg.com/browse/homebridge-lutron-caseta-smockle@3.0.6/.
